app/src/main/res/layout/multireddit_textview_white.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:paddingStart="18dp"
+    android:paddingEnd="18dp"
+    android:layout_height="48dp">
+
+    <TextView
+        android:id="@+id/name"
+        android:layout_width="wrap_content"
+        android:layout_height="48dp"
+        android:textColor="#fff"
+        android:gravity="center_vertical"
+        android:layout_alignParentBottom="true"
+        android:text="multireddit"
+        android:textSize="14sp"
+        />
+
+
+
+</RelativeLayout>
```

Comment:
It'd be worth running the android studio formatter on this and `drawer_loggedin` to get the 8 space property indents